features/list-style.yml

```diff
@@ -0,0 +1,114 @@
+name: List style
+description: The `list-style` CSS and the `list-style-image`, `list-style-position`, and `list-style-type` longhands style the marker for a list item.
```

Comment:
```suggestion
description: The `list-style` shorthand CSS property and the `list-style-image`, `list-style-position`, and `list-style-type` longhand properties set the position and appearance of a list item's marker.
```

Oh also here's an interesting style question that I don't think we've talked about before "longhands." MDN uses "constituent properties" — I wonder if we ought to do the same? Curious to have your thoughts on it, though I don't think we should resolve the question for everything in this PR.

Reply:
@ddbeck  Wanted to circle back on this- in talking with Miriam and doing some research, I would like to stick with longhands and shorthands over constituent properties. They seem to be the more common phrase- searching for CSS constituent properties, for instance, doesn't return useful information, but CSS shorthands do. Should this go in a styleguide?

Reply:
@jamesnw Yes, please. You can file an issue or open a PR to cover longhand/shorthand terminology. 👍

Reply:
PR open in https://github.com/web-platform-dx/web-features/pull/1891